pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -668,6 +669,7 @@ func (lbaas *LbaasV2) EnsureLoadBalancerDeleted(clusterName string, service *api
 
 		for _, pool := range poolsList {
 			poolIDs = append(poolIDs, pool.ID)
+            monitorIDs = append(monitorIDs, pool.MonitorID)
```

Comment:
1. fix the whitespace problem locally
2. (squash the commits, unless you used --amend directly)
3. git push -f to your fork.  This PR will magically be replaced with your new version.

(for larger / in-progress reviews, don't squash the review-driven commits until we're ready to approve the overall PR - that way we can re-review the incremental changes more easily)